### ISSUE TITLE
Local Music Playback

### ISSUE THREAD

**Issue by luqmanfadlli** [revblaze/AppleMusicUltra#19]
Do you have any plans to play local music files in the future? I love this beautifully designed app

**Comment by drexkooo** [comment]
+1

Don't really care about apple music, but I'd like to go back to iTunes era album thumbnails for my local library, and ESPECIALLY get rid of the full album view (eg: on "Recently added" i'd get a bunch of thumbnails for my albums, and if i clicked on one it would expand it to show the songs, rather then opening it in a separate screen where I need to click on a tiny arrow to go back to the recently added screen)

**Comment by Setloth** [comment]
yeah im really hoping that Apple themselves updates the Apple Music app cause its missing a lot of QoL features and design stuff but the local files are indeed a high note of the app. Wish we could have a perfect mix of all things
